app.py

```diff
@@ -1,12 +1,22 @@
+import flask
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    # return 'Hello World!'
+    items = session.get_items()
+    return render_template("index.html", items=items)
+
+@app.route('/', methods=['POST'])
```

Comment:
It would be nicer to use a descriptive path for this endpoint instead of just '/'

E.g. `'/add-item'`

Especially so things remain clear when there are multiple POST endpoints.